tests/db/migration_tests/test_v202509050_to_v202509100.py

```diff
@@ -37,6 +37,36 @@ async def test_add_created_to_resource_persistent_state(
     Check that the RPS column is added and that it is populated to the
     configuration model version that the resource first appears in.
     """
+    # Insert dangling rps entry
```

Comment:
I'd move this to the dump tool (and then regenerate the "pre" dump). so we can't make this same mistake again.